releases/v1.4.0-beta.toml

```diff
@@ -0,0 +1,92 @@
+
+### Client
+* **New registry configuration tooling for improved mirror and endpoint options** (https://github.com/containerd/containerd/pull/4138)
+* **Add Linux resources to spec options** (https://github.com/containerd/containerd/pull/4083)
+* **Use spec's mountLabel when mounting container rootfs** (https://github.com/containerd/containerd/pull/4051)
+* **Propagate snapshotter layer annotations on unpack** (https://github.com/containerd/containerd/pull/3911)
+* **Create image record after blob download to fix concurrent download issue** (https://github.com/containerd/containerd/pull/3972)
+* **Improve host fallback behaviour in docker remote** (https://github.com/containerd/containerd/pull/3868)
+* **Defer layer download until unpack to support skipping download when snapshotter reports already exists** (https://github.com/containerd/containerd/pull/3870)
+* **WithLease takes options for more flexibility** (https://github.com/containerd/containerd/pull/3719)
+* **Add spec options for host device and privileged handling** (https://github.com/containerd/containerd/pull/3718)
+* **Add unpack options for pull** (https://github.com/containerd/containerd/pull/3826)
+* **Split UID and GID namespace mapping in spec generation** (https://github.com/containerd/containerd/pull/3881)
+
+### API
+* **Add snapshot cleanup** (https://github.com/containerd/containerd/pull/3925)
+* **Add filters to snapshot list** (https://github.com/containerd/containerd/pull/3709)
+
+### Daemon
+* **Support NOTIFY_SOCKET for notifying the daemon's readiness to systemd** (https://github.com/containerd/containerd/pull/4088)
+
+### Windows
+* **Enable CRI plugin** (https://github.com/containerd/containerd/pull/4076)
+* **Add support for custom wcow and lcow sandbox scratch sizes** (https://github.com/containerd/containerd/pull/3965)
+* **Disk usage support in snapshotter** (https://github.com/containerd/containerd/pull/3785)
+
+### CRI
+
+* **Support CNI DNS capabilities** (https://github.com/containerd/cri/pull/1244)
+* **Build CRI Plugin on Windows and add presubmit** (https://github.com/containerd/cri/pull/1258)
+* **Use container annotations when creating containers** (https://github.com/containerd/cri/pull/1260)
+* **Add support for Windows containers process isolation** (https://github.com/containerd/cri/pull/1264)
+* **Add windows port forward support** (https://github.com/containerd/cri/pull/1284)
+* **Use `http` for localhost registry connections** (https://github.com/containerd/cri/pull/1328)
+* **Add `resolv.conf` to sandbox container mounts** (https://github.com/containerd/cri/pull/1344)
+* **Use host devices options from oci specs** (https://github.com/containerd/cri/pull/1349)
+* **Fix privileged supported** (https://github.com/containerd/cri/pull/1356)
+* **Add instrospection service** (https://github.com/containerd/cri/pull/1364)
+* **Unshare cgroup namespace for container** (https://github.com/containerd/cri/pull/1371)
```

Comment:
This is specific to cgroup v2

Reply:
I'm not sure what the correct wording here is.
`Unshare cgroupv2 namespace for container`
or
`Unshare cgroup namespace for container when using cgroupv2`
?

Reply:
the latter one please